src/components/Episodes/index.js

```diff
@@ -1,6 +1,8 @@
 import data from "./data.js";
+import { PREV_EPISODE, NEXT_EPISODE, SET_ACTIVE_INDEX, SET_OVERLAY_ACTIVE } from "./actions.js";
```

Comment:
ahhh, is this what you were talking about? I _think_ you can do `import * as ACTIONS from "./actions.js";` here.